proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -24753,6 +24738,7 @@ lemma preemption_path_ct_ready_if_schedulable:
    apply (rename_tac tcb, case_tac "tcb_state tcb"; clarsimp)
   apply (rule hoare_if)
    apply (wpsimp wp: check_budget_ct_ready_if_schedulable)
+
```

Comment:
Is this newline intentional?

Reply:
Nope. I fix it in the next PR though, so I'd prefer to merge this one immediately followed by the next, if no one minds